tests/test_run_reference_vmm.py

```diff
@@ -33,6 +27,57 @@ def process_exists(pid):
         return True
 
 
+def default_busybox_bzimage():
+    return s3_download(
+        resource_type="kernel",
+        resource_name="bzimage-hello-busybox",
+        first=True
+    )
+
+
+def default_busybox_elf():
+    return s3_download(
+        resource_type="kernel",
+        resource_name="vmlinux-hello-busybox",
+        first=True
+    )
+
+
+def default_ubuntu_bzimage():
+    return s3_download(
+        resource_type="kernel",
+        resource_name="bzimage-focal",
+        first=True
+    )
+
+
+def default_ubuntu_elf():
+    return s3_download(
+        resource_type="kernel",
+        resource_name="vmlinux-focal",
+        first=True
+    )
+
+
+def default_disk():
+    return s3_download(
+        resource_type="disk",
+        resource_name="ubuntu-focal-rootfs.ext4",
+        first=True
+    )
+
+
+KERNELS_INITRAMFS = [
```

Comment:
Can we add a comment here, when the tests are moved to a class, it's better to perform this inside a `setUp` function.

Reply:
I am not sure I understand what you mean. I don't think these defines will move to a class because they are the parameters of the tests. We were discussing on a separate issue I think about moving the actions you can perform on a `VM` in a class, not the tests.

Reply:
I believe `pytest` allow - module level `setUp` and `cleanUp` (I forgot the names). The way it is right now, the functions could get called even when someone `import`s this file. (Not that it matters in this case, because nobody is going to `import` the file.) But a cleaner and readable way is to actually perform actions like resource downloads inside a `setUp` function. 

I saw there was a comment about converting the test cases to a class, I thought it might be a good idea to take that up as a class level `setUp` or perhaps module level `setUp`. So just adding a FIXME or TODO for now is good enough.